arch/arm/soc/arm/musca_a/Kconfig.series

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2018 Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_SERIES_MUSCA
+	bool "ARM v2m MUSCA MCU Series"
+	depends on ARM
```

Comment:
`depends on ARM `is not, really, used in other SOC series definitions (other than ARM SOCs)

Reply:
fixed